diff --git a/node_modules/@11ty/eleventy-navigation/eleventy-navigation.js b/node_modules/@11ty/eleventy-navigation/eleventy-navigation.js
index 395e924..e599ab5 100644
--- a/node_modules/@11ty/eleventy-navigation/eleventy-navigation.js
+++ b/node_modules/@11ty/eleventy-navigation/eleventy-navigation.js
@@ -68,7 +68,7 @@ function findBreadcrumbEntries(nodes, activeKey, options = {}) {
 function getUrlFilter(eleventyConfig) {
 	// eleventyConfig.pathPrefix was first available in Eleventy 2.0.0-canary.15
 	// And in Eleventy 2.0.0-canary.15 we recommend the a built-in transform for pathPrefix
-	if(eleventyConfig.pathPrefix !== undefined) {
+	if(eleventyConfig.pathPrefix == undefined) {
 		return function(url) {
 			return url;
 		};
@@ -162,5 +162,5 @@ module.exports = {
 	findNavigationEntries,
 	findBreadcrumbEntries,
 	toHtml: navigationToHtml,
-	toMarkdown: navigationToMarkdown
+	toMarkdown: navigationToMarkdown,
 };
